executor/aggregate.go

```diff
@@ -375,9 +371,11 @@ func (w *HashAggPartialWorker) run(ctx sessionctx.Context, waitGroup *sync.WaitG
 			w.shuffleIntermData(sc, finalConcurrency)
 		}
 		w.memTracker.Consume(-w.chk.MemoryUsage())
-		waitGroup.Done()
 	}()
 	for {
+		if w.hashAgg.stats != nil {
+			atomic.AddInt64(&w.hashAgg.stats.PartialTaskNum, 1)
```

Comment:
Does `partialTaskNum` mean the chks that partial workers received?

Reply:
I think `partialTaskNum` means the partialworker Indicates how many times the `updatePartialResult` function has been executed.